impacket/krb5/kerberosv5.py

```diff
@@ -112,7 +112,7 @@ def getKerberosTGT(clientName, password, domain, lmhash, nthash, aesKey='', kdcH
     reqBody['rtime'] = KerberosTime.to_asn1(now)
     reqBody['nonce'] =  random.getrandbits(31)
 
-    # Yes.. this shouldn't happend but it's inherited from the past
+    # Yes.. this shouldn't of happened but it's inherited from the past
```

Comment:
This one is wrong.. 'shouldn't happen' is the right phrase

Reply:
I'll fix this one myself don't worry.

Reply:
Want me to fix this ?